src/components/button/Button.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { TouchableOpacity, Text } from 'react-native';
+import { styles } from './Button.styles';
+
+export const Button = ({title = 'Button', isDisabled = false, ...rest}) => {
+    let buttonStyle;
+
+    if (isDisabled) {
+        buttonStyle = [styles.buttonEnabled, styles.buttonDisabled];
```

Comment:
you can improve this. for enable set only enabled, for disabled disabled.

Reply:
I have changed this to use `buttonContainer` with the difference being applied with `buttonDisabled` and `buttonEnabled`